database/schema.sql

```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
     PRIMARY KEY (image_id),
     FOREIGN KEY (region_id) REFERENCES region(region_id),
     FOREIGN KEY (constellation_id) REFERENCES constellation(constellation_id)
-);
\ No newline at end of file
+);
+
+INSERT INTO aurora_colour (aurora_colour_id, colour, description, meaning) VALUES
+(1, 'Green', 'No significant activity', 'Aurora is unlikely to be visible by eye or camera from anywhere in the UK.'),
+(2, 'Yellow', 'Minor geomagnetic activity', 'Aurora may be visible by eye from Scotland and may be visible by camera from Scotland, northern England and Northern Ireland.'),
+(3, 'Amber', 'Amber alert: possible aurora', 'Aurora is likely to be visible by eye from Scotland, northern England and Northern Ireland; possibly visible from elsewhere in the UK. Photographs of aurora are likely from anywhere in the UK.'),
+(4, 'Red', 'Red alert: aurora likely', 'It is likely that aurora will be visible by eye and camera from anywhere in the UK.');
+
+INSERT INTO region (region_name, latitude, longitude) VALUES
+('Northern Ireland', 54.61, -6.62),
+('Scotland', 56.49, -4.20),
+('Wales', 52.13, -3.78),
+('North East', 54.97, -1.61),
+('North West', 53.78, -2.70),
+('Yorkshire and the Humber', 53.80, -1.54),
+('East Midlands', 52.90, -1.23),
+('West Midlands', 52.48, -1.89),
+('East of England', 52.24, 0.90),
+('London', 51.51, -0.13),
+('South East', 51.28, -0.78),
+('South West', 50.78, -3.79);
+
+INSERT INTO county (county_name, latitude, longitude, region_id) VALUES
+('Antrim', 54.71, -6.22, 1),
```

Comment:
Similar for all counties, this accuracy will be important for weather data later in the project.